suite-native/module-settings/src/screens/SettingsCoinEnablingScreen.tsx

```diff
@@ -1,28 +1,47 @@
 import { useDispatch, useSelector } from 'react-redux';
 import { useCallback } from 'react';
+import { View } from 'react-native';
 
 import { useFocusEffect } from '@react-navigation/native';
+import { LinearGradient } from 'expo-linear-gradient';
 
 import { Screen, ScreenSubHeader } from '@suite-native/navigation';
-import { Translation, useTranslate } from '@suite-native/intl';
+import { useTranslate } from '@suite-native/intl';
 import { BtcOnlyCoinEnablingContent, DiscoveryCoinsFilter } from '@suite-native/coin-enabling';
-import { Box, Text } from '@suite-native/atoms';
+import { Box } from '@suite-native/atoms';
 import { selectIsBitcoinOnlyDevice } from '@suite-common/wallet-core';
 import {
     selectDiscoverySupportedNetworks,
     selectEnabledDiscoveryNetworkSymbols,
     setIsCoinEnablingInitFinished,
 } from '@suite-native/discovery';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { hexToRgba } from '@suite-common/suite-utils';
+
+const GRADIENT_HEIGHT = 40;
+
+const gradientBaseStyleParams = prepareNativeStyle<{ bottom: number }>((_, { bottom }) => ({
+    width: '100%',
+    height: GRADIENT_HEIGHT,
+    position: 'absolute',
+    bottom,
```

Comment:
This fixes the issue with toggle buttons below the gradient.
```suggestion
    pointerEvents: 'none',
    bottom,
```

Reply:
Thanks 🙏 , I am about to look into that later today, thanks for testing and suggestions :)

Reply:
I did it right away in the end:
b53f863ef7